CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Every user-facing change should have an entry in this changelog. Please respect
 
 ## Unreleased
 - [Improvement] Make it possible to override k8s resources in plugins using `k8s-override` patch. (by @foadlind)
+- [Bugfix] Fix a race condition that could prevent a newly provisioned Studio container from starting due to a FileExistsError when creating logs directory.
```

Comment:
As indicated at the top of the changelog file, please don't add entries here. Instead, create new changelog entries with `make changelog-entry`. See: https://docs.tutor.overhang.io/tutor.html#contributing

Reply:
I'll merge as-is and fix the changelog manually.

Reply:
Thanks, @regisb and sorry, I didn't get to fixing this. I'll keep this in mind.